hoextdown/hoextdown_fuzzer.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+dict = hoextdown.dict
```

Comment:
If you rename the dict file to match the name of the fuzzer binary (or vise versa) you won't need the options file. 
(just fewer things to maintain)